builder.go

```diff
@@ -1,15 +1,19 @@
 package lifecycle
 
 import (
+	"fmt"
+
+	"github.com/BurntSushi/toml"
+
+	"github.com/buildpacks/lifecycle/api"
+
```

Comment:
Fix import grouping. `fmt` should be group with other stdlib imports at the top of the `import` block. `github.com/buildpacks/lifecycle/api` should be group with other `"github.com/buildpacks/lifecycle/*` imports at the bottom and `github.com/BurntSushi/toml` should be in a separate group in the middle.

Reply:
Sorry, I thought that `make all` is doing that for me.
Why don't we do it?
Anyway, I fixed the import order in all of the files, please let me know if I missed something.
Thanks!

Reply:
I made sure stdlib, external, and internal imports were all in separate groups, but not correctly (it created three new groups instead of having only three groups total)... It has some unfortunately edge cases depending on the state of the input :/

Adding similar settings to your IDEA can help to avoid getting into the edge cases